docs/v3/group-chat.md

```diff
@@ -0,0 +1,165 @@
+# Group Chat
+
+## Summary
+
+* XMTP V3 is introducing Group Chat based on the MLS [specification](https://www.rfc-editor.org/rfc/rfc9420.html).
+* A group can be anywhere from 1 person to 20,000.
```

Comment:
@insipx whenever we figure out the limit we should update this number.

Reply:
Added suggestion below